### PR TITLE
add graphacademy promos and ad

### PR DESCRIPTION
**Pull request by adam-cowley**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-981.surge.sh

Updated pages:
- [LOAD CSV](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/clauses/load-csv)
- [Examples](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/constraints/examples)
- [Constraints](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/constraints)
- [Syntax](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/constraints/syntax)
- [GenAI integrations](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/genai-integrations)
- [Indexes](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/indexes)
- [Index hints for the Cypher planner](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/indexes/search-performance-indexes/index-hints)
- [Create, show, and delete indexes](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/indexes/search-performance-indexes/managing-indexes)
- [Search-performance indexes](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/indexes/search-performance-indexes/overview)

File: modules/ROOT/pages/constraints/examples.adoc [hunk 1]
```diff
@@ -1,7 +1,8 @@
-:description: Examples of how to manage constraints used for ensuring data integrity.
-:page-toclevels: 1
 [[constraints-examples]]
 = Examples
+:description: Examples of how to manage constraints used for ensuring data integrity.
+:page-toclevels: 1
+include::{ga-courses-uri}/cypher-indexes-constraints/ad.adoc[]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/constraints/examples.adoc:5]: In the preview, this goes here: https://graphacademy.neo4j.com/courses/app-go/?ref=docs-ad-go - shouldnt it go to the Indexes & Constraints course?

File: modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc [hunk 2]
```diff
@@ -9,11 +9,14 @@ _Node vector indexes were released as a public beta in Neo4j 5.11 and general av
...
 Vector indexes enable similarity searches and complex analytical queries by representing nodes or properties as vectors in a multidimensional space.
 
-The following resources provide hands-on tutorials for working with LLMs and vector indexes in Neo4j:
+// The following resources provide hands-on tutorials for working with LLMs and vector indexes in Neo4j:
+
+// * link:https://graphacademy.neo4j.com/courses/llm-fundamentals/[GraphAcademy -> LLM & Neo4j Fundamentals]
+// * link:https://graphacademy.neo4j.com/courses/llm-vectors-unstructured/[GraphAcademy -> Introduction to Vector Indexes and Unstructured Data]
+// * link:https://neo4j.com/docs/genai/tutorials/embeddings-vector-indexes/[GenAI documentation -> Embeddings & Vector Indexes Tutorial]
+
+include::{ga-promo-uri}/llms.adoc[]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc:19]: The old version with just bullet points looks better, I think.

File: modules/ROOT/pages/introduction/index.adoc [hunk 3]
```diff
@@ -27,6 +27,8 @@ This introduction will cover the following topics:
 * xref:introduction/cypher-aura.adoc[]
 
 
+include::{ga-promo-uri}/{ga-promo}.adoc[]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/introduction/index.adoc:30]: There are now 2 mentions of GraphAcademy on this page.

**Review by JPryce-Aklundh — COMMENTED** [review]
Where the left-side ToC is long, you have to scroll down to see the promo (e.g on LOAD CSV page).

all include::{ga-courses-uri}... directs to here: https://graphacademy.neo4j.com/courses/app-go/?ref=docs-ad-go.  Shouldnt they go to the specific courses? e.g. The ad on the Constraints pages take me to the "Building Neo4j Applications with Go"

**Comment by adam-cowley** [comment]
> Where the left-side ToC is long, you have to scroll down to see the promo (e.g on LOAD CSV page).
> 
> all include::{ga-courses-uri}... directs to here: https://graphacademy.neo4j.com/courses/app-go/?ref=docs-ad-go. Shouldnt they go to the specific courses? e.g. The ad on the Constraints pages take me to the "Building Neo4j Applications with Go"

I will double-check, but [the file seems to have the correct link in there](https://github.com/neo4j-graphacademy/courses/blob/main/asciidoc/courses/cypher-indexes-constraints/ad.adoc)

File: modules/ROOT/pages/introduction/index.adoc (hunk 3 above)
Comment (adam-cowley) [reply modules/ROOT/pages/introduction/index.adoc:30]: There should be more?

If it is a point of contention, I'm happy to remove the `Additionally...` sentence

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-981.surge.sh

Updated pages:
- [LOAD CSV](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/clauses/load-csv)
- [Examples](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/constraints/examples)
- [Constraints](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/constraints)
- [Syntax](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/constraints/syntax)
- [GenAI integrations](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/genai-integrations)
- [Indexes](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/indexes)
- [Index hints for the Cypher planner](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/indexes/search-performance-indexes/index-hints)
- [Create, show, and delete indexes](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/indexes/search-performance-indexes/managing-indexes)
- [Search-performance indexes](https://neo4j-docs-cypher-981.surge.sh/cypher-manual/5/indexes/search-performance-indexes/overview)

File: modules/ROOT/pages/introduction/index.adoc (hunk 3 above)
Comment (JPryce-Aklundh) [reply modules/ROOT/pages/introduction/index.adoc:30]: No, I think it was fine as it was.

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request closed without merging**